WSFrontend/org.openl.rules.ruleservice.ws/src/org/openl/rules/ruleservice/storelogdata/CacheAndWriteOutputStream.java

```diff
@@ -0,0 +1,65 @@
+package org.openl.rules.ruleservice.storelogdata;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.OutputStream;
+
+import org.apache.cxf.io.CachedOutputStream;
+
+public class CacheAndWriteOutputStream extends CachedOutputStream {

```

Comment:
Add Javadocs with description for which purposes is this class needed and what it does.